pkg/api/v1/types/types.go

```diff
@@ -50,13 +50,20 @@ func (c *ConditionCreate) NewCondition(kind ptypes.ConditionKind) *ptypes.Condit
 
 // ConditionUpdate is the request payload to update an existing condition.
 type ConditionUpdate struct {
-	ID              uuid.UUID             `json:"id"`
+	ConditionID     uuid.UUID             `json:"condition_id"`
+	TargetID        uuid.UUID             `json:"target_id"`
```

Comment:
TargetID here would be the object being updated, so the Server UUID in our case?

Could we stick to camelcase for the struct JSON tag values here